### ISSUE TITLE
Can't start the app

### ISSUE THREAD

**Issue by sntll** [autokey/autokey#613]
## Classification:

Bug.

## Reproducibility:

Always.

## Version

AutoKey version: autokey-0.95.4

Used GUI:
GTK.

Installed via:
pip

Linux Distribution:
Artix LXDE.

## Summary

After an apparent successful installation, when I click the shortcut menu an error message appears and the app is not initiated.

## Steps to Reproduce (if applicable)

- Run `pip install autokey`

## Expected Results

- The app would start.

## Actual Results

- Instead, the popup message `Invalid desktop entry file: '/home/a1/.local/share/applications/autokey-gtk.desktop'.` appears.

Debugging
```
~$ pip install autokey
Defaulting to user installation because normal site-packages is not writeable
Collecting autokey
  Using cached autokey-0.95.4-py3-none-any.whl (228 kB)
Requirement already satisfied: dbus-python in ./.local/lib/python3.9/site-packages (from autokey) (1.2.16)
Requirement already satisfied: python-xlib in /usr/lib/python3.9/site-packages (from autokey) (0.31)
Requirement already satisfied: pyinotify in /usr/lib/python3.9/site-packages (from autokey) (0.9.6)
Requirement already satisfied: six>=1.10.0 in /usr/lib/python3.9/site-packages (from python-xlib->autokey) (1.16.0)
Installing collected packages: autokey
Successfully installed autokey-0.95.4
```

I had to add `/home/a1/.local/bin` to `$PATH` because a warning appeared the first time I installed it.

```
$ autokey-gtk --verbose
Traceback (most recent call last):
  File "/home/a1/.local/bin/autokey-gtk", line 5, in <module>
    from autokey.gtkui.__main__ import main
  File "/home/a1/.local/lib/python3.9/site-packages/autokey/gtkui/__main__.py", line 4, in <module>
    from autokey.gtkapp import Application
  File "/home/a1/.local/lib/python3.9/site-packages/autokey/gtkapp.py", line 45, in <module>
    from autokey.gtkui.notifier import get_notifier
  File "/home/a1/.local/lib/python3.9/site-packages/autokey/gtkui/notifier.py", line 23, in <module>
    gi.require_version('AppIndicator3', '0.1')
  File "/usr/lib/python3.9/site-packages/gi/__init__.py", line 126, in require_version
    raise ValueError('Namespace %s not available' % namespace)
ValueError: Namespace AppIndicator3 not available
```

Also, there's no Autokey folder in `/home/.config`

**Comment by josephj11** [comment]
A few comments:
Probably just a typo in your comment, but the folder should be `$HOME/.config/autokey` - under your user and lower case a.
I don't know if it will make a difference, but try again using
`pip3 --user install autokey`
It looks like leaving the `--user` flag out may have had other consequences.
See [this](https://github.com/autokey/autokey/wiki/Installing#arch-linux).
(Just noticed the wiki links to the aur packages are broken. I noted it to the rest of the team.) 
Also, especially in Arch land, you should have our latest production version - 0.95.10.
I believe it's packaged in aur. There should be two packages - one for each front end (Arch weirdness), so pick the one you want.

To keep things simple, uninstall before each new install (until it works right.)

**Comment by sntll** [comment]
Thanks for the help,

The pip3 gives me the following message:
```
$ sudo pip3 --user install autokey

Usage:   
  pip3 <command> [options]

no such option: --user
```
I installed the version 0.95.10 by running:
```
$ sudo pip install https://github.com/autokey/autokey/archive/refs/tags/v0.95.10.zip
Collecting https://github.com/autokey/autokey/archive/refs/tags/v0.95.10.zip
  Using cached https://github.com/autokey/autokey/archive/refs/tags/v0.95.10.zip
Requirement already satisfied: pyinotify in /usr/lib/python3.9/site-packages (from autokey==0.95.10) (0.9.6)
Requirement already satisfied: python-xlib in /usr/lib/python3.9/site-packages (from autokey==0.95.10) (0.31)
Requirement already satisfied: six>=1.10.0 in /usr/lib/python3.9/site-packages (from python-xlib->autokey==0.95.10) (1.16.0)
Building wheels for collected packages: autokey
  Building wheel for autokey (setup.py) ... done
  Created wheel for autokey: filename=autokey-0.95.10-py3-none-any.whl size=342610 sha256=2c57b836d36777323330d2f14489970004781ce458ab602c4314ee33ad33f81b
  Stored in directory: /tmp/pip-ephem-wheel-cache-o714vyld/wheels/d1/5a/93/ff9c3945cd42b7287254d073e642f0a08453d2b47cf627944d
Successfully built autokey
Installing collected packages: autokey
Successfully installed autokey-0.95.10

```
The popup error message disappeared, but Autokey is still not launching.
```
$ autokey-gtk --verbose
bash: /home/a1/.local/bin/autokey-gtk: No such file or directory
```

**Comment by BlueDrink9** [comment]
Should be `pip3 install --user autokey-gtk` (no `sudo`)

**Comment by sntll** [comment]
Sorry for the late reply,

This is the result:
```
$ pip3 install --user autokey-gtk
ERROR: Could not find a version that satisfies the requirement autokey-gtk
ERROR: No matching distribution found for autokey-gtk
```
It installs without the `-gtk`
```
~$ pip3 install --user autokey
Collecting autokey
  Using cached autokey-0.95.4-py3-none-any.whl (228 kB)
Requirement already satisfied: python-xlib in /usr/lib/python3.9/site-packages (from autokey) (0.31)
Requirement already satisfied: dbus-python in ./.local/lib/python3.9/site-packages (from autokey) (1.2.16)
Requirement already satisfied: pyinotify in /usr/lib/python3.9/site-packages (from autokey) (0.9.6)
Requirement already satisfied: six>=1.10.0 in /usr/lib/python3.9/site-packages (from python-xlib->autokey) (1.16.0)
Installing collected packages: autokey
Successfully installed autokey-0.95.4
```
However, the popup error message appears again: `Invalid desktop entry file: '/home/a1/.local/share/applications/autokey-gtk.desktop'`

Edit: I also tried `$ pip install https://github.com/autokey/autokey/archive/refs/tags/v0.95.10.zip` without `sudo` and the same error happens:
```
$ pip install https://github.com/autokey/autokey/archive/refs/tags/v0.95.10.zip
Defaulting to user installation because normal site-packages is not writeable
Collecting https://github.com/autokey/autokey/archive/refs/tags/v0.95.10.zip
  Using cached https://github.com/autokey/autokey/archive/refs/tags/v0.95.10.zip
Requirement already satisfied: pyinotify in /usr/lib/python3.9/site-packages (from autokey==0.95.10) (0.9.6)
Requirement already satisfied: python-xlib in /usr/lib/python3.9/site-packages (from autokey==0.95.10) (0.31)
Requirement already satisfied: six>=1.10.0 in /usr/lib/python3.9/site-packages (from python-xlib->autokey==0.95.10) (1.16.0)
Building wheels for collected packages: autokey
  Building wheel for autokey (setup.py) ... done
  Created wheel for autokey: filename=autokey-0.95.10-py3-none-any.whl size=343314 sha256=16aa21c169319b27f3c70103217a975405626953e30f1c4b4e1ef14aafd36562
  Stored in directory: /tmp/pip-ephem-wheel-cache-lwhhzqg2/wheels/d1/5a/93/ff9c3945cd42b7287254d073e642f0a08453d2b47cf627944d
Successfully built autokey
Installing collected packages: autokey
Successfully installed autokey-0.95.10
```

```
$ autokey-gtk --verbose
Traceback (most recent call last):
  File "/home/a1/.local/bin/autokey-gtk", line 5, in <module>
    from autokey.gtkui.__main__ import main
  File "/home/a1/.local/lib/python3.9/site-packages/autokey/gtkui/__main__.py", line 4, in <module>
    from autokey.gtkapp import Application
  File "/home/a1/.local/lib/python3.9/site-packages/autokey/gtkapp.py", line 45, in <module>
    from autokey.gtkui.notifier import get_notifier
  File "/home/a1/.local/lib/python3.9/site-packages/autokey/gtkui/notifier.py", line 23, in <module>
    gi.require_version('AppIndicator3', '0.1')
  File "/usr/lib/python3.9/site-packages/gi/__init__.py", line 126, in require_version
    raise ValueError('Namespace %s not available' % namespace)
ValueError: Namespace AppIndicator3 not available
```

**Comment by sntll** [comment]
Good news,

I tried to manually build the package from AUR, but another error happened with every package necessary to install Autokey, and it had these characteristics:
```
error: could not prepare transaction
error: failed to commit transaction (conflicting files)
libname: /insert/file/name/here exists in filesystem
Errors occurred, no packages were upgraded.
```
I installed `pamac-gtk` and from there I was finally able to install `autokey-gtk 0.95.10-4`.
I also would like to express my gratitude for the creation and support behind this fantastic program.
Thank you very much.
